static/src/javascripts/projects/common/modules/discussion/loader.js

```diff
@@ -38,7 +40,9 @@ define([
     DiscussionApi,
     CommentBox,
     Comments,
+    frontend,
```

Comment:
Can we call this `discussion-frontend` or something else? Didn't initially realise it was the `discussion-frontend` module.